openmc_plotter/plotmodel.py

```diff
@@ -104,6 +107,10 @@ def __init__(self):
         self.ids = None
         self.instances = None
 
+        # ID and Props (curren)
+        self.__ids_map = None
+        self.__props_map = None
```

Comment:
What was the reasoning for these not being normal attributes? (i.e., why the leading double underscore?)

Reply:
They aren't used outside this class. / They weren't initially attributes so they aren't expected to be used outside.